lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/directories/Directories.java

```diff
@@ -0,0 +1,12 @@
+package org.enso.desktopenvironment.directories;
+
+import java.nio.file.Path;
+
+public interface Directories {
```

Comment:
Do you want other code outside of this module to implement this interface? If not, then prevent it:
- (in old Java days) I'd recommend `abstract class` with _package private_ constructor
- now there is also an option to defined `sealed` interface

Reply:
good old Java days :cowboy_hat_face: